wpeverest/everest-forms
# Fix - now correctly closes label if no desc arg is passed

### All Submissions:






### Changes proposed in this Pull Request:



Closes #335  .

### How to test the changes in this Pull Request:

1. Try passing empty desc arg when rendering a field type `radio`.
2. Notice the label is now correctly closed, and no longer enclosed within `ul` tags.

### Types of changes:




### Other information:




### Changelog entry

> Fix - now correctly closes label if no desc arg is passed

#### Before
![image](https://user-images.githubusercontent.com/30156167/71868646-26ac2680-3137-11ea-959d-9ad7c74ec06f.png)

#### After
![image](https://user-images.githubusercontent.com/30156167/72137100-4c912f80-33b2-11ea-9c38-154519f86937.png)
